qa/tasks/ses_qa.py

```diff
@@ -178,6 +179,125 @@ def _get_drive_group_limit(self, **kwargs):
                            if 'deepsea' in x and 'drive_group' in x['deepsea'])
         return int(drive_group['deepsea']['drive_group']['custom']['data_devices']['limit'])
 
+    def ses_rack_dc_region_unavailability(self, **kwargs):
+        """
+        Simulates rack, DC and region unavailability by 
+        modifying Ceph crushmap
+        """
+        self.scripts.run(
+                self.master_remote,
+                'ses_rack_dc_region_unavailability.sh',
+                )
+    
+    def ses_network_failure(self, **kwargs):
+        """
+        Simulates network failure using tc netem tool
+        """
+        self.scripts.run(
+                self.master_remote,
+                'ses_network_failure.sh',
+                )
+
+    def ses_cephfs_test_mount(self, **kwargs):
+        #mon_hosts = self.role_lookup_table['mon'].values()
+        self.scripts.run(
+                self.master_remote,
+                'ses_cephfs_test_mount.sh',
+                #args=mon_hosts,
```

Comment:
drop this line

Reply:
dropped 63005ed621156d98fab9598cdb710eca081f8f46